src/hooks/useLogDownloader/index.ts

```diff
@@ -142,6 +150,8 @@ const useLogDownloader = (
           });
           setIsLoading(false);
         });
+    } else if (url === null) {
+      setError("Log URL not specified, unable to download.");
```

Comment:
I think i might tweak the wording here to something along the lines of this:
```suggestion
      setError("Could not retrieve log URL from task, unable to download.");
```